CHANGELOG.md

```diff
@@ -2,7 +2,19 @@
 
 librdkafka v1.9.3 is a maintenance release:
 
- *  Self-contained static libraries can now be built on Linux arm64 too
+ * Self-contained static libraries can now be built on Linux arm64 too
+ * Fixes to the transactional and idempotent producer (#3971).
+
+
+## Fixes
+
+### Transactional producer fixes
+
+ * When requesting an abort with drain and bump after message timeouts, the bump was executed first, causing a series of retries on InitProducerIdRequest until a transaction timeout happen and the producer was fenced. This commit changes the order of the operations, starting the drain bump after the complete transaction ack (#3971).
+ * While a commit operation was in queue, a timeout happens that can cause an abort. The state changes from COMMITTING_TRANSACTION to ABORTABLE_ERROR to ABORTING_TRANSACTION. When the broker comes up the error is permanent or fatal because the state has changed from the initial one (#3971).
```

Comment:
it's not clear from this wording what was existing behavior and what you changed.